src/Csharp/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set_target_properties(
 		DHARTAPICSharp
 		PROPERTIES
 		VS_DOTNET_REFERENCES "${normal_references}"
-		VS_DOTNET_TARGET_FRAMEWORK_VERSION "v4.5"
+		VS_DOTNET_TARGET_FRAMEWORK_VERSION "v4.8"
```

Comment:
I updated the build instructions to mention this .net version issue https://github.com/cadop/dhart/pull/43

I would prefer to wait a little on incrementing this framework version until I can test the built dll's in different contexts (e.g. Unity).  

Could you revert the cmake changes for this PR?

Reply:
Changed in the latest commit